packages/nvidia_nat_mcp/src/nat/plugins/mcp/auth/auth_provider.py

```
@@ -422,8 +479,10 @@ async def _nat_oauth2_authenticate(self, user_id: str | None = None) -> AuthResu
                 logger.info(f"Initialized token storage with object store '{self._token_storage_object_store_name}'")
             except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)